src/screens/OtherProfile/index.js

```diff
@@ -200,19 +200,28 @@ const OtherProfile = () => {
       await generateAnonProfile();
     }
   };
+  console.log(isAnonimity, 'nakal');
 
   const __renderListHeader = () => {
     const __renderBlockIcon = () => {
       if (isBlocking)
         return (
-          <View style={styles.buttonFollowing}>
-            <Text style={styles.textButtonFollowing}>Blocked</Text>
+          <View style={styles.buttonFollowing(isAnonimity)}>
+            <Text style={styles.textButtonFollowing(isAnonimity)}>Blocked</Text>
           </View>
         );
 
       return (
-        <View style={{...styles.btnMsg, borderColor: colors.gray1}}>
-          <BlockIcon width={20} height={20} style={{color: colors.gray1}} />
+        <View
+          style={{
+            ...styles.btnMsg(isAnonimity),
+            borderColor: isAnonimity ? COLORS.anon_primary : COLORS.signed_primary
+          }}>
+          <BlockIcon
+            width={20}
+            height={20}
+            style={{color: isAnonimity ? COLORS.anon_primary : COLORS.signed_primary}}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk seem to be related to the introduction of a new color scheme based on user anonymity. The logic seems fine, but it's important to ensure that these color changes do not negatively affect the accessibility of the application. For example, make sure there is enough contrast between text and background colors for readability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->